src/lib/fetch.ts

```diff
@@ -138,7 +141,10 @@ async function _handleRequest(
   let result: any
 
   try {
-    result = await fetcher(url, requestParams)
+    result = await fetcher(url, {
+      ...requestParams,
+      cache: 'no-cache', // UNDER NO CIRCUMSTANCE SHOULD THIS OPTION BE REMOVED, YOU MAY BE OPENING UP A SECURITY HOLE IN NEXT.JS APPS
```

Comment:
can we also include a link to their docs? (https://nextjs.org/docs/app/building-your-application/caching#opting-out-1)